contao/languages/de/tl_calendar.xlf

```diff
@@ -26,6 +26,10 @@
                 <source>Application Secret</source>
                 <target>Application Secret</target>
             </trans-unit>
+            <trans-unit id="tl_calendar.facebookEvents_accessToken.0">
+                <source>Access Token</source>
+                <target>Zugriffsschlüssel</target>
+            </trans-unit>
```

Comment:
Add a explanation translation `tl_calendar.facebookEvents_accessToken.1` with the link how to create a never expires access token.